### ISSUE TITLE
Can RBM set price of extensions?

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#1218]
I'm not sure if RBM already does this. Can RBM increase the price of my rig mid-rental if the market pumps so that extensions cost more?

**Comment by RainbowMiner** [comment]
The mrr rig's price are continuously adapted to the market, even during rentals. The question is, will an extension be billed, using the new price or the initial price of the rental?

**Comment by samdave123** [comment]
Yes that is the question, do you not know? I think it is currently pricing extensions at the initial price of the rental which can be really underpriced if there's a pump, but maybe I'm wrong on that. If it's working how I think it's working, then we're losing a lot of profit, especially if our max rental time is long.

**Comment by astronyu** [comment]
Based on what I seen, extension will still be billed at the same initial price of the rental.

**Comment by samdave123** [comment]
If RBM cannot increase the price of extensions, then can it disable extensions if profitability increases?

**Comment by samdave123** [comment]
I've just set     "AllowExtensions": "0" until my extensions can be prevented from being underpriced.

**Comment by RainbowMiner** [comment]
Actually, that automatic extension disable, in case the rig profitability would grow above a certain percentage sounds like a smart idea to me.

**Comment by samdave123** [comment]
> 
> 
> Actually, that automatic extension disable, in case the rig profitability would grow above a certain percentage sounds like a smart idea to me.

Is that doable with the MRR API? If so, yes please it will boost our profits!
